stdlib/public/core/Filter.swift

```diff
@@ -108,8 +108,7 @@ public struct LazyFilterIndex<Base : Collection> : Comparable {
 /// Returns `true` iff `lhs` is identical to `rhs`.
 @warn_unused_result
 public func == <Base : Collection>(
-  lhs: LazyFilterIndex<Base>,
-  rhs: LazyFilterIndex<Base>
+  lhs: LazyFilterIndex<Base>, rhs: LazyFilterIndex<Base>
```

Comment:
The codebase converges on the other style here, I think.  Once we start breaking out parameters to a new line, we put each on a separate line.